lib/devos/configGenerators.nix

```diff
@@ -0,0 +1,84 @@
+{ lib, ... }: let
```

Comment:
Let's think about it: top-level, function instantiation only depends on `lib`.
This returns functions or function generators (where more dependencies are in need to be injected).

Same goes for the importers, which does not have any additional deps at all